modules/product/src/Plugin/Field/FieldWidget/ProductVariationAttributesWidget.php

```diff
@@ -96,57 +104,6 @@ public static function isApplicable(FieldDefinitionInterface $field_definition)
   /**
    * {@inheritdoc}
    */
-  public static function defaultSettings() {
-    return [
-      'attribute_widget_type' => 'select',
```

Comment:
There's also the schema in attribute_widget_type.
We also need to re-export core.entity_form_display.commerce_line_item.product_variation.add_to_cart.yml to remove the now-gone setting.